desktop-app/app/components/WebView/screenshotUtil.js

```diff
@@ -8,13 +8,17 @@ import {promisify} from 'util';
 import Promise from 'bluebird';
 import path from 'path';
 import fs from 'fs-extra';
+import Jimp from 'jimp';
 import PromiseWorker from 'promise-worker';
 import NotificationMessage from '../NotificationMessage';
 import {userPreferenceSettings} from '../../settings/userPreferenceSettings';
 import {type Device} from '../../constants/devices';
 import {captureOnSentry} from '../../utils/logUtils';
+import {SCREENSHOT_MECHANISM} from '../../constants/values';
+import mutexify from 'mutexify/promise';
```

Comment:
you need to add `mutexify` to `package,json`

Reply:
Yeah, not sure how it got missed. Will add it.